src/ui/public/directives/rows.js

```diff
@@ -18,15 +18,33 @@ module.directive('kbnRows', function ($compile, $rootScope, getAppState, Private
         if (_.isNumeric(contents)) $cell.addClass('numeric-value');
 
         const createAggConfigResultCell = function (aggConfigResult) {
-          const $cell = $(document.createElement('td'));
+          const $cell = $(
+            `<td>
+              <span class="table-cell-filter">
+                <span
+                  ng-click="clickHandler($event, false)"
+                  class="fa fa-search-plus"
+                  tooltip="Filter for value"
+                  tooltip-append-to-body="1"
+                ></span>
+
+                <span
+                  ng-click="clickHandler($event, true)"
+                  class="fa fa-search-minus"
+                  tooltip="Filter out value"
+                  tooltip-append-to-body="1"
+                ></span>
+              </span>
+            </td>`
+          );
+
           const $state = getAppState();
           const clickHandler = filterBarClickHandler($state);
           $cell.scope = $scope.$new();
           $cell.addClass('cell-hover');
-          $cell.attr('ng-click', 'clickHandler($event)');
-          $cell.scope.clickHandler = function (event) {
+          $cell.scope.clickHandler = function (event, negate) {
             if ($(event.target).is('a')) return; // Don't add filter if a link was clicked
-            clickHandler({ point: { aggConfigResult: aggConfigResult } });
+            clickHandler({ point: { aggConfigResult: aggConfigResult }, negate: negate });
```

Comment:
The [style guide](https://github.com/elastic/kibana/blob/fd6fd6411e2fa4aff8bec098943f8bac31d70163/style_guides/js_style_guide.md) suggests using the property value shorthand syntax (simply `negate` instead of `negate: negate`).